x-pack/plugins/ml/public/application/explorer/explorer_charts/components/explorer_chart_label/explorer_chart_label.js

```diff
@@ -48,16 +49,30 @@ export function ExplorerChartLabel({
   const entityFieldBadges = entityFields.map((entity) => {
     const key = `${infoTooltip.chartFunction}-${entity.fieldName}-${entity.fieldType}-${entity.fieldValue}`;
     return (
-      <Fragment key={`badge-wrapper-${key}`}>
-        <ExplorerChartLabelBadge entity={entity} />
+      <EuiFlexGroup gutterSize="none" alignItems="center" key={`badge-wrapper-${key}`}>
+        <EuiFlexItem grow={false}>
+          {mode === 'embeddable' ? (
+            <EuiText size="xs">
+              <EuiTextColor color={'success'} component={'span'}>
+                {`(${entity.fieldName}: ${entity.fieldValue})`}
+              </EuiTextColor>
+            </EuiText>
+          ) : (
+            <ExplorerChartLabelBadge entity={entity} />
+          )}
+        </EuiFlexItem>
+
         {onSelectEntity !== undefined && (
-          <EntityFilter
-            onFilter={applyFilter}
-            influencerFieldName={entity.fieldName}
-            influencerFieldValue={entity.fieldValue}
-          />
+          <EuiFlexItem grow={false}>
```

Comment:
This probably isn't a new issue, but for long field names / values, the filter icons can end up being rendered out of the visible area:

<img width="1269" alt="Screenshot 2024-11-07 at 13 14 06" src="https://github.com/user-attachments/assets/827e6ec5-ba08-418a-8509-60246cea919a">

Reply:
This probably needs a separate issue - happens on `main` and also affects the charts when in the Anomaly Explorer.

Reply:
I was able to improve it with truncating and adding in a tooltip for the full entity value:
<img width="650" alt="image" src="https://github.com/user-attachments/assets/977e1730-fbb3-4b86-b0df-6630915b846d">

Reply:
Looks like this needs some more tweaking. The anomaly charts look good for long values, but not for short text values there can be a large gap:

<img width="458" alt="Screenshot 2024-11-08 at 10 16 14" src="https://github.com/user-attachments/assets/09b56a00-f285-460c-91b1-470cf604dd0e">

Reply:
Latest changes look good for shorter field name / values. Separate issue created to improve for long values: https://github.com/elastic/kibana/issues/199549